app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -165,6 +166,14 @@ def display_phone_to_deliver_to
     end
   end
 
+  def resend_otp_code_path
+    otp_send_path(otp_delivery_selection_form:
+    {
```

Comment:
nit: this formatting is throwing me off with the brace on the next line, how do you feel about:

```ruby
otp_send_path(otp_delivery_selection_form: {
  otp_method: @delivery_method,
  resend: true
})
```